bluebottle/tasks/tests/test_mails.py

```diff
@@ -40,8 +40,10 @@ def test_member_applied_to_task_mail(self):
 
         # Task owner receives email about new task member
         self.assertEquals(len(mail.outbox), 1)
-        self.assertNotEquals(mail.outbox[0].body.find("applied for your task"),
-                             -1)
+        body = mail.outbox[0].body
+        self.assertTrue('applied for your task' in body)
+        self.assertTrue(self.task_member.member.first_name in body)
+        self.assertTrue(self.task_member.member.last_name in body)
```

Comment:
I think the BluebottleUserFactory creates objects with first_name=user_1 and last_name=user_1, these test might not prove that you're not using short_name.